larq_compute_engine/mlir/python/converter.py

```diff
@@ -52,6 +53,8 @@ def _contains_training_quant_op(graph_def):
 def convert_keras_model(
     model: tf.keras.Model,
     *,  # Require remaining arguments to be keyword-only.
+    inference_input_type: tf.DType = tf.float32,
+    inference_output_type: tf.DType = tf.float32,
```

Comment:
I don't know what they do in TFL, but it seems that we could probably drop the `inference_` from the argument names here? I don't really have a strong preference though.

Reply:
I don't have a strong preference either, but I went for the `inference_` prefix since [TFLite does the same](https://www.tensorflow.org/api_docs/python/tf/lite/TFLiteConverter#attributes), but I'd be happy to just use `input_type` and `output_type` instead since we also don't adhere to the type conventione the TFLite converter uses here.

Reply:
I also don't have a preference. This part of the code is where training connects to inference-only, so writing `inference_` sort-of shows that its meant for the final inference-only graph and not for the training graph. That might be the rationale behind it.